File: Week01/Even_Odd/Bitwise.cpp
#include<iostream>

using namespace std;


bool isEven(int n) {
    if ((n & 1) == 0)
        return true;
    else 
        return false;
}


int main() {
    cout << "Enter an integer to check even odd: ";
    int x;
    cin >> x;

    if (isEven(x))
        cout << "EVEN number";
    else
        cout << "ODD number";
    
    return 0;

}

File: Week01/Even_Odd/Bitwise.cs
using static System.Console;

class Bitwise
{
    public static bool IsEven(int n)
    {
        if ((n & 1) == 0)
            return true;
        else
            return false;
    }


    static void Main()
    {
        WriteLine("Enter an integer: ");
        int x = int.Parse(ReadLine());

        if (IsEven(x))
            WriteLine("EVEN number");
        else
            WriteLine("ODD number");
    }
}